twins/mocks/service.go

```diff
@@ -38,6 +41,15 @@ func CreateDefinition(channels []string, subtopics []string) twins.Definition {
 	return def
 }
 
+// CreateTwin creates twin
+func CreateTwin(channels []string, subtopics []string) twins.Twin {
+	id++
```

Comment:
I do not know if having a global var would be a good idea here. I do not go in deeper analysis, but there is some "code smell". I would prefer some internal structure, created in the `main()` via `New()` fnc, then passed back here... Or something like this.

Reply:
This `id` is not a global var, it's a package level var, not exported, used for the sole purpose of generating unique id's for the mock entitites that we use to test service. It's rather so called closure variable. The function users can't know anything about it.

Reply:
You can probably use this UUID mock here: https://github.com/mainflux/mainflux/blob/master/pkg/uuid/mock.go

Reply:
Looks like you already have it here https://github.com/mainflux/mainflux/blob/e8852a16ec984045094612ae16512d5c8f082885/twins/mocks/service.go#L24

Reply:
@darkodraskovic sorry I did not see this was for the mock. I will approve then. I let you take a look at this Manu's remark optionally, but this is good to me.

Reply:
@manuio i purposefully didn't use the mock uuid generator, cause i don't want to handle uuid error creation. I really need only mock id's, can be really anything, it's just for the purpose of differentiating the twins, so I just give them unique serial number like this - no possibility of error, so no need for error handling.